benchmarks/transport/ghex_p2p_bi_cb_avail_mt.cpp

```diff
@@ -151,7 +152,13 @@ int main(int argc, char *argv[])
                 make_zero(rmsgs[j]);
             }
 
-            comm.barrier();
+#ifdef USE_OPENMP
+#pragma omp single
+#endif
+            barrier.rank_barrier(comm);
```

Comment:
why do we need the `pragma omp single` here? Can a general barrier work here?

Reply:
I tried reproduce the behavior of previous implementation, so Iwould keep the omp single here